### ISSUE TITLE
HTTP Status Codes has lost its usefulness after the new theme

### ISSUE THREAD

**Issue by swantzter** [tfredrich/RestApiTutorial.com#69]
I've been referencing https://www.restapitutorial.com/httpstatuscodes for a long time now due to the fact that it's shown all HTTP status codes in an overview you have not needed to scroll:
This has been a great strength compared to Wikipedia or IETF which the page references

![bild](https://github.com/tfredrich/RestApiTutorial.com/assets/10356719/a36a3894-3320-47ed-9ebe-a7ca32a58922)

After the redesign I suddenly have to scroll and it's hard to get just a quick overview, would it be possible to add a quick reference sheet/cheat-sheet again?

**Comment by mike-brady** [comment]
Agreed. They new look is form over function. As a reference page, the old layout was perfect. It was very quick and easy to see everything and find what you need.

**Comment by long-eared-owl** [comment]
As a quick fix I put an old HTML vanilla version on my GitHub page:
[https://long-eared-owl.github.io/httpstatuscodes/](https://long-eared-owl.github.io/httpstatuscodes/)
It also has a one side print css.

**Comment by tfredrich** [comment]
Indeed, thanks for the feedback. I'll see what I can do!

**Comment by tfredrich** [comment]
Per @long-eared-owl 's https://long-eared-owl.github.io/httpstatuscodes/ was able to embed raw HTML and style it. Hopefully, this will work for us.

**Comment by swantzter** [comment]
@tfredrich yeah, that looks much better.

Only slight thing now is that the stars need a different colour when in dark mode, right now it's always #000, should probably be #fff when the theme is dark

https://github.com/tfredrich/RestApiTutorial.com/commit/2e20037430951b2aa863109073f553a334b0f1fd#diff-1cd81fe79bc601a40159f0a27f6f10d5ed044f297eea88db73bf82f2bfb0b33eR44

**Comment by CoreyKovalik** [comment]
At first, I actually thought the website disappeared! I agree, the old view was very useful and far easier to scan & parse with your eyes, IMHO.
